CHANGELOG.md

```diff
@@ -2,7 +2,14 @@
 
 ## next
 
+* Added `include_nil` and `include_unset` options to `View` for refining the rendering of nil or unset values respectively. These can be set when defining the view with `Blueprint.view`, and default to false. For example: `view :default, include_unset: nil { ... }`.
+  * `include_nil`: include attributes with `nil` values on the object (as checked with `@object.key?`).
```

Comment:
* I'd change it to "...with `nil` values on the underlying data object that the Blueprint is wrapping".
* the (as checked with ...) is in the wrong line...it should be in the one below (include_unset)